### DISCUSSION TITLE
How can i fix this keymap conflict with Blink and LSP

### DISCUSSION THREAD

**Discussion by neonvoidx** [LazyVim/LazyVim#5131] [Q&A]
https://github.com/LazyVim/LazyVim/blob/2d7a04be47b94010fd6c1a2be0283c0d4e5738ae/lua/lazyvim/plugins/lsp/keymaps.lua#L24
Here `<c-k>` is being mapped to signature help for LSP, but I want it to be prev_item for blink completion selection, is there a way to keep both mappings, currently im trying to use `<c-j>` and `<c-k>` to move up/down respectively for Blink's completion menu,<C-j> works but `<C-k>` i just get `No signature help available`

Blink config override:
```
return {
  "saghen/blink.cmp",
  opts = {
    keymap = {
      ["<C-k>"] = { "select_prev", "fallback" },
      ["<C-j>"] = { "select_next", "fallback" },

```

**Comment by neonvoidx** [comment]
I think I got it:
```
return {
  "neovim/nvim-lspconfig",
  opts = function(_, opts)
    local keys = require("lazyvim.plugins.lsp.keymaps").get()
    keys[#keys + 1] = { "<C-k>", false, mode = { "i" } }

    opts.servers.eslint.settings.workingDirectory = { mode = "auto" }
    opts.inlay_hints = { enabled = false }
    opts.setup.eslint = function()
      require("lazyvim.util").lsp.on_attach(function(client)
        if client.name == "eslint" then
          client.server_capabilities.documentFormattingProvider = true
        elseif client.name == "tsserver" then
          client.server_capabilities.documentFormattingProvider = false
        end
      end)
      vim.cmd([[autocmd BufWritePre *.tsx,*.ts,*.jsx,*.js EslintFixAll]])
    end
    return opts
  end,
}

```

would be nice if both worked somehow, but maybe not possible? so I just disabled the keymap

**Reply by fernandoaleman** [reply]
@jacobrreed this worked for me. Thanks for posting. 👍

**Comment by Weissle** [comment]
keys[#keys + 1] = { "<C-k>", false, mode = { "i" } }
mode = {"i"} is important. Missing this will not disable the keymap.

**Comment by t1gu1** [comment]
For others, you got a nice and simple example in the [Doc](https://www.lazyvim.org/plugins/lsp#%EF%B8%8F-customizing-lsp-keymaps) 😁
